client/errorReporter.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { window } from 'vscode';
+
+export interface MlxprsError {
+    popupMessage: string;
+    reportedMessage: string;      // module URI on MarkLogic
```

Comment:
These comments are a little surprising - `reportedMessage` above is an error message, I don't see a module URI being present.

Reply:
My "copy & paste" oversight.